config.json

```
@@ -620,7 +620,7 @@
         "name": "Raindrops",
         "uuid": "f77ac2d1-cf3a-497d-bf04-b484a5a9cb37",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": ["objects", "array-transformations", "strings"],
```

I would set the prerequisites here looking at these types of solutions: https://github.com/exercism/javascript/blob/main/exercises/practice/raindrops/.meta/proof.ci.js
People can always use more fancy stuff if they want what with what is in there you can write a good solution for the exercise and there are far better exercises to use array transformation concepts on than this one.
Also the old "topics" seem to go into that direction.

```suggestion
        "prerequisites": ["arithmetic-operators", "strings", "conditionals"],
```

---

I get what you're saying, let's make the prerequisites of this one easier :+1: